bundles/org.openhab.binding.modbus.helioseasycontrols/README.md

```diff
@@ -0,0 +1,306 @@
+| humidityControlSteps             | humidityControl | Humidity control steps (in percent)                              | Number:Dimensionless     | %    | 5 - 20                                                                                               |
+| humidityStopTime                 | humidityControl | Humidity stop time in hours (0-24)                               | Number:Time              | h    | 0 - 24                                                                                               |
+| co2ControlStatus                 | co2Control      | CO2 control status                                               | Number                   |      | 0 = off, 1 = stepped, 2 = stepless                                                                   |
+| co2ControlSetValue               | co2Control      | CO2 control set value (in ppm)                                   | Number:Dimensionless     | ppm  | 300 - 2000                                                                                           |
+| co2ControlSteps                  | co2Control      | CO2 control steps (in ppm)                                       | Number:Dimensionless     | ppm  | 50 - 400                                                                                             |
+| vocControlStatus                 | vocControl      | VOC control status                                               | Number                   |      | 0 = off, 1 = stepped, 2 = stepless                                                                   |
+| vocControlSetValue               | vocControl      | VOC control set value (in ppm)                                   | Number:Dimensionless     | ppm  | 300 - 2000                                                                                           |
+| vocControlSteps                  | vocControl      | VOC control steps (in ppm)                                       | Number:Dimensionless     | ppm  | 50 - 400                                                                                             |
+| comfortTemp                      | unitConfig      | Comfort Temperature                                              | Number:Temperature       | °C   | 10.0 - 25.0                                                                                          |
+| timeZoneDifferenceToGmt          | general         | Time Zone Difference to GMT                                      | Number:Time              | h    | -12 - 14                                                                                             |
+| dateFormat                       | general         | Date format                                                      | Number                   |      | 0 = dd.mm.yyyy, 1 = mm.dd.yyyy, 2 = yyyy.mm.dd                                                       |

```

Comment:
The user can specify the date format on its own in the sitemap or items configuration. Is this Channel necessary?

Reply:
That's again how it's defined in the device's Modbus protocol specification...

Reply:
It's good practice that your binding internally uses always the same date scheme when communicating with the device. If the binding provides the date as `DateTime`, it's the users's choice how to format it.